semgrep/semgrep/formatter/sarif.py

```diff
@@ -185,15 +190,6 @@ def format(
             https://docs.oasis-open.org/sarif/sarif/v2.1.0/cs01/sarif-v2.1.0-cs01.html
         """
 
-        # Check each rule match for any fixes
-        rule_matches, rule_matches_fixes = tee(rule_matches)
-        fixes = [
-            self._rule_match_to_sarif_fix(rule_match)
-            for rule_match in rule_matches_fixes
-        ]
-        # Filter out rule matches w/no fixes
-        fixes = list(filter(None, fixes))
-
         output_dict = {
             "$schema": "https://raw.githubusercontent.com/oasis-tcs/sarif-spec/master/Schemata/sarif-schema-2.1.0.json",
```

Comment:
Can we update this broken link to https://docs.oasis-open.org/sarif/sarif/v2.0/csprd01/schemas/sarif-schema.json ?

Reply:
Will do.